.eslintrc.js

```diff
@@ -19,7 +19,12 @@ module.exports = {
     sourceType: "module",
   },
   rules: {
-    "@typescript-eslint/no-non-null-assertion": 0,
     "@typescript-eslint/no-explicit-any": 0,
+    "@typescript-eslint/no-empty-interface": [
+      "error",
+      {
+        allowSingleExtends: true,
```

Comment:
There's value in my opinion in creating what is effectively an interface alias in a library.  Especially when generic are involved and allowing for meaningful names on interfaces that may later be extended.